.circleci/config.yml

```diff
@@ -302,7 +302,7 @@ jobs:
       - run:
           name: Build Notifications Panel
           command: |
-            NODE_ENV=production npx lerna run build --scope='@automattic/notifications' -- -- --output-path=$CIRCLE_ARTIFACTS/notifications-panel
+            cd apps/notifications/ && NODE_ENV=production yarn run build --output-path=$CIRCLE_ARTIFACTS/notifications-panel
```

Comment:
For my edification... why do we `cd` here instead of using `yarn workspace`?

Reply:
Personal preference...
Both commands work the same (afaik) and cd'ing is easier to read in my opinion.